contrib/dracut/90zfs/mount-zfs.sh.in

```diff
@@ -77,7 +80,7 @@ if import_pool "${ZFS_POOL}" ; then
 			# decrypt them
 			TRY_COUNT=5
 			while [ $TRY_COUNT != 0 ]; do
-				zfs load-key "${ZFS_DATASET}"
+				zfs load-key "${ZFS_CRYPTPARENT}"
```

Comment:
it's used here

Reply:
it still only runs when the loop has executed, but I'll move it anyway.